File: app/src/main/java/com/yhjoo/dochef/model/MultiItemRecipe.java
package com.yhjoo.dochef.model;

import com.chad.library.adapter.base.entity.MultiItemEntity;

public class MultiItemRecipe implements MultiItemEntity {
    int itemType;
    Recipe content;
    String pager_title;

    public MultiItemRecipe(int itemType, Recipe content) {
        this.itemType = itemType;
        this.content = content;
    }

    public MultiItemRecipe(int itemType, String title) {
        this.itemType = itemType;
        this.pager_title = title;
    }

    public MultiItemRecipe(int itemType) {
        this.itemType = itemType;
    }

    public String getPager_title() {
        return pager_title;
    }

    public Recipe getContent() {
        return content;
    }

    @Override
    public int getItemType() {
        return itemType;
    }
}

File: app/src/main/java/com/yhjoo/dochef/data/model/MultiItemRecipe.kt
package com.yhjoo.dochef.data.model

import com.chad.library.adapter.base.entity.MultiItemEntity

class MultiItemRecipe : MultiItemEntity {
    var multiType: Int
    var content: Recipe? = null
    var pagerTitle: String? = null

    constructor(itemType: Int, content: Recipe?) {
        this.multiType = itemType
        this.content = content
    }

    constructor(itemType: Int, title: String?) {
        this.multiType = itemType
        pagerTitle = title
    }

    constructor(itemType: Int) {
        this.multiType = itemType
    }

    override fun getItemType(): Int {
        return multiType
    }
}